### PR TITLE
updated density of WC

### PR DESCRIPTION
**Pull request by katie-taylor**
Changed WC density according to maximum theoretical density of sintered WC

### CODE REVIEW

**Comment by shimwell** [comment]
Thanks for the addition, it is much better than the density in the library at the moment.

I notice 15.67 appears to be the theoretical density which is slightly different to the density achieved in the paper

File: neutronics_material_maker/data/structural_materials.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "WC": {
         "chemical_equation": "WC",
-        "density": 18.0,
+        "density": 15.67,
+        "comment": "https://www.birmingham.ac.uk/Documents/college-eps/xmat/Low-Temperature-Spark-Plasma-Sintering-of-Pure-Nano-WC-Powder.pdf",
```
Comment (shimwell) [inline neutronics_material_maker/data/structural_materials.json]:
```suggestion
        "comment": "Theoretical density as specified in https://www.birmingham.ac.uk/Documents/college-eps/xmat/Low-Temperature-Spark-Plasma-Sintering-of-Pure-Nano-WC-Powder.pdf",
```

**Comment by shimwell** [comment]
Thanks for the addition happy to merging this one

We will need another PR from develop to main and a version bump for it to get into the PyPi version

**Comment by katie-taylor** [comment]
Should I do this? If so how?

**Comment by shimwell** [comment]
I have just made the PR over here https://github.com/ukaea/neutronics_material_maker/pull/145

**Pull request merged**